cmd/kubernetes_show.go

```diff
@@ -11,6 +11,8 @@ import (
 	"github.com/spf13/cobra"
 )
 
+var stableVersion = "1.22.2"
```

Comment:
Rather than using a hardcoded version, can you fetch all the versions and check which one is stable in there? Probably check what `civo k3s versions` is doing because it already outputs non-deprecated versions.

Reply:
`civo k3s versions` only outputs this - 

```
+-------------+--------+---------+
| Version     | Type   | Default |
+-------------+--------+---------+
| 1.22.2-k3s1 | stable | true    |
+-------------+--------+---------+
```